File: Src/AssemblyInfoCommon.cs
using System;
using System.Reflection;
using System.Resources;

// this is used to version artifacts. AssemblyInformationalVersion should use semantic versioning (http://semver.org/)
// be sure to update the corresponding VB.NET AssemblyInfoCommon.vb
[assembly: AssemblyInformationalVersion("3.0.0-beta")]
[assembly: AssemblyVersion("3.0.0.0")]

[assembly: AssemblyCompany("Kent Boogaart")]
[assembly: AssemblyProduct("KBCsv")]
[assembly: AssemblyCopyright("© Copyright. Kent Boogaart.")]
[assembly: AssemblyTrademark("")]
[assembly: AssemblyCulture("")]
[assembly: CLSCompliant(true)]
[assembly: NeutralResourcesLanguage("en-US")]

#if DEBUG
[assembly: AssemblyConfiguration("Debug")]
#else
[assembly: AssemblyConfiguration("Release")]
#endif

File: Src/AssemblyInfoCommon.vb
Imports System
Imports System.Reflection
Imports System.Resources
Imports System.Runtime.InteropServices

' this is used to version artifacts. AssemblyInformationalVersion should use semantic versioning (http://semver.org/)
' be sure to update the corresponding C# AssemblyInfoCommon.cs
<Assembly: AssemblyInformationalVersion("2.0.0-beta")> 
<Assembly: AssemblyVersion("2.0.0.0")> 

<Assembly: AssemblyCompany("Kent Boogaart")> 
<Assembly: AssemblyProduct("KBCsv")> 
<Assembly: AssemblyCopyright("© Copyright. Kent Boogaart.")> 
<Assembly: AssemblyTrademark("")> 
<Assembly: AssemblyCulture("")> 
<Assembly: CLSCompliant(True)> 
<Assembly: ComVisible(False)> 
<Assembly: NeutralResourcesLanguage("en-US")> 

#If DEBUG Then
<Assembly: AssemblyConfiguration("Debug")>
#Else
<Assembly: AssemblyConfiguration("Release")> 
#End If